packages/expo-ui/ios/PickerView.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2025-present 650 Industries. All rights reserved.
+
+import SwiftUI
+import ExpoModulesCore
+
+class PickerProps: ExpoSwiftUI.ViewProps {
+  @Field var options: [String] = []
+  @Field var selectedIndex: Int?
+  @Field var variant: String?
+  var onOptionSelected = EventDispatcher()
+}
+
+struct PickerView: ExpoSwiftUI.View {
+  @State var selection: Int?
+  @EnvironmentObject var props: PickerProps
+
+  var body: some View {
+    if #available(iOS 17.0, *) {
```

Comment:
Can we display a warning when `Picker` isn't available?

Reply:
Sure, will add it in a followup